notebooks/ts_generic_example.ipynb

```diff
@@ -213,18 +214,22 @@
     "            # Add a brief description to help users understand the purpose or functionality of this trained model.\n",
     "            description=\"ADD_DESCRIPTION_HERE\",\n",
     "        ),\n",
+    "        data_processing_parameters=TrainDataProcessingParametersProto(\n",
+    "            input_columns=[0, 1, 2, 3],\n",
+    "            output_columns=[4]\n",
+    "        )\n",
     "    ),\n",
     "    inputs=MLTrainInputsProto(\n",
     "        # Provide the specific dataset storage ID of the data you uploaded to TQ42.\n",
-    "        data=DatasetStorageInfoProto(storage_id=\"ENTER_DATASET_STORAGE_ID_HERE\")\n",
+    "        data=DatasetStorageInfoProto(storage_id=\"1023200d-6d36-45ea-b66d-afbcc192eabc\")\n",
```

Comment:
Please also revert this line so people know they need to fill something in here.

Reply:
did that also in all the other notebooks